- git is only needed when 'molecules' and 'crystals' aren't supplied by the caller

--- CMakeLists.txt.orig	2024-02-10 14:49:49 UTC
+++ CMakeLists.txt
@@ -26,8 +26,8 @@ set(AvogadroLibs_VERSION
 set(AvogadroLibs_VERSION_PATCH   "0")
 set(AvogadroLibs_VERSION
   "${AvogadroLibs_VERSION_MAJOR}.${AvogadroLibs_VERSION_MINOR}.${AvogadroLibs_VERSION_PATCH}")
-find_package(Git)
-determine_version(${AvogadroLibs_SOURCE_DIR} ${GIT_EXECUTABLE} "AvogadroLibs")
+#find_package(Git)
+#determine_version(${AvogadroLibs_SOURCE_DIR} ${GIT_EXECUTABLE} "AvogadroLibs")
 
 option(BUILD_SHARED_LIBS "Build with shared libraries" ON)
 
